webapp/loader.py

```diff
@@ -0,0 +1,27 @@
+from webapp import create_app, db
+from model import FinancialData
+from parsing_csv import parsing_csv
+
+app = create_app()
+
+my_list = []
+
+
+def save_data(filename):
```

Comment:
save_data это самое популярное и самое непонятное имя для функции сохранения данных в БД. Хотелось бы понимать, какие именно данные сохраняет функция, и куда она их сохраняет. Возможно у тебя получится придумать лучше, чем у меня.
```suggestion
def insert_parsed_csv(financial_data: dict):
```

Reply:
поправил